script.js

```diff
@@ -95,8 +93,7 @@ function incrementScore(scoreSpan) {
     scoreSpan.innerText = parseInt(scoreSpan.innerText) + 1;
 }
 
-// not working yet
-// function cleanUpBottom(){
-// history.removeChild([history.children[9],history.children[10]])
-// history.removeChild(history.children[12])
-// }
+function cleanUpBottom(){
+    history.removeChild(history.children[12])
+    history.removeChild(history.children[12])
```

Comment:
So the goal here is to only show the last 5 pairs of results, right?

This `12` jumps out to me as a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)). In that link (which you should read), you'll find examples of it.

But it's generally bad because while it works, it:
- Doesn't self-document why this number was chosen/what it means
- Makes it difficult to update logic or keep it coherent around the codebase

So in this instance, I might suggest something like the following:
```ts
const maxHistoryPairs = 5;

function cleanUpBottom() {
  const historyPairToRemoveIndex = (maxHistoryPairs + 1) * 2;
  // Note: I'm (Matt) personally not sure why `12` works here.
  // I would've expected children 0-9 to be the 5 * 2 pairs that we want, and indexes 10 * 11
  // to be the ones we want to remove, not 12 & 13?
  // Is there something else in the DOM here clogging up `history`'s children?

  history.removeChild(history.children[historyPairToRemoveIndex]); // Removes first
  history.removeChild(history.children[historyPairToRemoveIndex]); // Removes second, now unshifted up from prior removal
}
```

Reply:
So the reason why it's 12 is because the actual Score Numbers are part of the "results" `div` (which is the `history`) I should actually try to separate those out..